modules/cnv-subscribing-to-hco-catalog.adoc

```diff
@@ -12,28 +12,36 @@ namespace access to the {CNVProductName} Operators.
 
 .Prerequisites
 
-* `OperatorGroup` and `CatalogSource` Custom Resource Definition objects (CRDs),
-both created in the `openshift-cnv` namespace
+* Create a project called `openshift-cnv`.
 
 .Procedure
 
 . Open a browser window and navigate to the {product-title} web console.
 
 . Navigate in the web console to the *Operators → OperatorHub* page.
 
-. Scroll or type a keyword into the *Filter by keyword* box to find the KubeVirt
-HyperConverged Cluster Operator, then click on it.
+. Scroll or type a keyword into the *Filter by keyword* box to find the
+{CNVProductNameStart} Operator, then click on it.
 
 . Read the information about the Operator and click *Install*.
 
 . On the *Create Operator Subscription* page:
-.. Select one of the following:
+.. Select *A specific namespace on the cluster* from the *Installation Mode*
+list and choose the `openshift-cnv` namespace.
++
+[NOTE]
+====
 *** *All namespaces on the cluster (default)* installs the Operator in the default
 `openshift-operators` namespace to watch and be made available to all namespaces
 in the cluster. This option is not always available.
```

Comment:
This option is not always available or supported maybe? Please double check with @rthallisey maybe.

Reply:
Good call, @alexxa 
@rthallisey do you think it makes sense to change this to just say:
"This option is not supported for use with container-native virtualization." ? I don't think the "This option is not always available" part is very helpful anyway.

Reply:
I think we want to limit the deploy to `openshift-cnv` since it's how QE tested and some of CNV's operator are singletons.

Reply:
@rthallisey Okay, so then the way it is currently phrased seems correct. The step instructs the user to "Select *A specific namespace on the cluster* from the *Installation Mode*
list and choose the `openshift-cnv` namespace." It describes the two options but says that the "all namespaces" option is not supported for use with CNV. 

Or did you mean that the other installation mode option is going to be removed from the UI?

Reply:
I would say to only document the specific namespace method, so it's clear that's the path we want to steer the user.